src/main/java/eod/Gameboard.java

```diff
@@ -83,6 +83,30 @@ public void removeObject(int x, int y) throws IllegalArgumentException {
         board[x][y] = null;
     }
 
+    public BoardPosition getPosition(Player player, Point p) {
+        if(player.isPlayerA()) {
+            if(p.x < firstLine) {
+                return BoardPosition.SELF_BASE;
+            } else if(p.x < middle) {
+                return BoardPosition.SELF_CONFLICT;
+            } else if(p.x < secondLine) {
+                return BoardPosition.ENEMY_CONFLICT;
+            } else {
+                return BoardPosition.ENEMY_BASE;
+            }
+        } else {
+            if(p.x >= secondLine) {
+                return BoardPosition.SELF_BASE;
+            } else if(p.x >= middle) {
+                return BoardPosition.SELF_CONFLICT;
+            } else if(p.x >= firstLine) {
+                return BoardPosition.ENEMY_CONFLICT;
+            } else {
+                return BoardPosition.ENEMY_BASE;
+            }
```

Comment:
Maybe we can integrate those position lines with BoardPosition?

Reply:
I don't think so cuz SELF_BASE means differently to the two players.

Reply:
Alright, that makes sense.